src/dao/AccountDao.java

```diff
@@ -42,7 +43,7 @@ public AccountDao() {
      * @throws dao.DaoException if there is an error connecting to the database
      * or executing the query
      */
-    public Object getAccount(int accountId) throws DaoException {
+    public Account getAccount(int accountId) throws DaoException {
```

Comment:
Need to implement this. The parameter should also change to a string since that's what the Account class has as the identifier.